x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/data_types_definition.tsx

```diff
@@ -249,6 +300,12 @@ export const TYPE_DEFINITION: { [key in DataType]: DataTypeDefinition } = {
     }),
     value: 'long_range',
   },
+  ip_range: {
+    label: i18n.translate('xpack.idxMgmt.mappingsEditor.dataType.ipRangeDescription', {
+      defaultMessage: 'IP range',
+    }),
+    value: 'ip_range',
```

Comment:
Maybe this deserves a description too?

Reply:
I think this can probably be added when we address any copy changes for the `range` data type.

Reply:
That makes sense!